### PR TITLE
[[ Documentation ]] Fixes to lockMessages.lcdoc

### PR DESCRIPTION
**Pull request by asayd**
- Added messages tag.
- Added terms to references.
- Added links to referenced tokens.

### CODE REVIEW

File: docs/dictionary/property/lockMessages.lcdoc [hunk 1]
```diff
@@ -31,26 +31,26 @@ from being <trigger|triggered>,or to speed up operations when
 If the <lockMessages> <property> is set to true, the following are not
 sent: 
 
-        * Navigation messages (such as openCard, <closeStack>, and so
```
Comment (montegoulding) [inline docs/dictionary/property/lockMessages.lcdoc:34]: These will render as code I think

**Comment by montegoulding** [comment]
👍 
@livecode-vulcan review ok e19339e

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok e19339e256a58c2868343c83ede36ce29d30ac06

**Comment by livecode-vulcan** [comment]
😎 test success e19339e256a58c2868343c83ede36ce29d30ac06
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/93)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/93)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/93)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/93)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/93)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/93)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/93)

**Pull request merged**